src/hover-provider/entry.ts

```diff
@@ -11,13 +11,20 @@ export class HoverProvider {
       return null;
     }
 
-    const addonResults = await queryELSAddonsAPIChain(project.providers.hoverProviders, project.root, {
+    const internalResults = await queryELSAddonsAPIChain(project.builtinProviders.hoverProviders, project.root, {
       textDocument,
       position,
       results: [],
       server: this.server,
     });
 
+    const addonResults = await queryELSAddonsAPIChain(project.providers.hoverProviders, project.root, {
+      textDocument,
+      position,
+      results: internalResults,
+      server: this.server,
+    });
+
     if (addonResults.length) {
       return addonResults[0];
```

Comment:
Not sure how to approach that hover provider should return only hover, while we can get multiple from different addons. Should we somehow merge them?

Reply:
@godric3 good point, right now I don't have answer for it, looks like LS protocol limitation (multiple servers could provide hover details, but only one should be returned from server instance)

Reply:
addons should be smart enough to decide on it